### ISSUE TITLE
Accessibility : Post Settings TalkBack Improvements

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#10895]
For this audit the governing principles are as follows ([inspired by a previous audit](https://github.com/wordpress-mobile/WordPress-Android/issues/7288))

- All the clickable elements should be marked as so.
- All nonactionable and decorative elements should be non-focusable.
- Relevant content should be grouped together.
- All important elements should be described through content description.

### Talkback Audit 

* Post Settings
	* First option says "Status Draft in pager". Correct the pager reference. 
	* Second options reads "Publish Immediately, Imediately". One of the "immediately"s need to be removed.
	* Password,Categories, Tags,Slug, Location, Excerpt reads as"not set,not set" 
	* Excerpt even when set still says "not set". This issue correlates to the avoid so it should be addressed in one go.

**Comment by shiki** [comment]
For this batch:

- The repeated labels seem to be the most important
- The contrast changes, we _could_ do later as a separate batch with other contrast changes (if there are any)

Something that we can add:

- The _Categories & Tags_ and other related elements are not marked as “Heading“. We could use the [accessibilityHeading](https://developer.android.com/guide/topics/ui/accessibility/principles#headings_within_text) attribute for this even if it's only supported on API 28+. I think that's okay? The heading attribute is used when navigating by heading instead of the default.

**Comment by jd-alexander** [comment]
> For this batch
> The repeated labels seem to be the most important
> The contrast changes, we could do later as a separate batch with other contrast changes (if there are any)

Understood. A good approach to make reviews easier 👍 

> Something that we can add
> The Categories & Tags and other related elements are not marked as “Heading“. We could use the accessibilityHeading attribute for this even if it's only supported on API 28+. I think that's okay? The heading attribute is used when navigating by heading instead of the default.

This is a great addition! We can leverage the [AccessibilityDelegate](https://developer.android.com/reference/android/support/v4/view/AccessibilityDelegateCompat) for proper backward compatibility support when setting the headings. Will investigate usage as I go through the implementation.

**Comment by malinajirka** [comment]
>The “pager“ word being spoken is new. I wonder if this is due to the EditPostActivity refactoring, @malinajirka?

Thanks for the ping. I'm not aware of any changes which might be related to it. But you never know.
